tools/car_porting/auto_fingerprint.py

```diff
@@ -25,8 +24,7 @@
   parser.add_argument("platform", help="The platform, or leave empty to auto-determine using fuzzy", default=None, nargs='?')
   args = parser.parse_args()
 
-  route = Route(args.route)
-  lr = MultiLogIterator(route.qlog_paths())
+  lr = SegmentRangeReader(args.route, ReadMode.QLOG)
```

Comment:
I think some tools will still want to be able to specify the read mode. This tool doesn't need rlogs so the user shouldn't have to specify ```/q``` or whatever